packages/ETLoad.package/ETLOopMapTest.class/instance/testResolvingOfArbitraryOops.st

```diff
@@ -0,0 +1,21 @@
+testing
+testResolvingOfArbitraryOops
+	| oopMap |
+	
+	oopMap := ETLOopMap
+		newFromInternalObjects: (Dictionary with: 16r0 -> 123)
+		andOutPointers: { 99 }.
+
+	"small integer"
+	self assert: 42 equals: (oopMap resolve: 85 ifInvalid: [ self fail. ]).
```

Comment:
either add a comment explaining the 85 or write down the calculation `42 << 1 | 1`